Fakebook.Profile/Fakebook.Profile.RestApi/appsettings.json

```diff
@@ -1,10 +1,14 @@
 {
-  "Logging": {
-    "LogLevel": {
-      "Default": "Information",
-      "Microsoft": "Warning",
-      "Microsoft.Hosting.Lifetime": "Information"
-    }
-  },
-  "AllowedHosts": "*"
+    "ConnectionStrings": {
+        "BlobStorage": "dummy-value",
```

Comment:
a dummy value for the blob storage connection string is not needed, but this is especially misleading since the actual clode is looking for the connection string at `BlobStorage:ConnectionString`, not here.